pkg/cmd/container/inspect.go

```diff
@@ -40,20 +41,13 @@ func Inspect(ctx context.Context, client *containerd.Client, containers []string
 		OnFound: f.Handler,
 	}
 
-	var errs []error
-	for _, req := range containers {
-		n, err := walker.Walk(ctx, req)
-		if err != nil {
-			errs = append(errs, err)
-		} else if n == 0 {
-			errs = append(errs, fmt.Errorf("no such container: %s", req))
+	err := walker.WalkAll(ctx, containers, true)
+	if len(f.entries) > 0 {
+		if formatErr := formatter.FormatSlice(options.Format, options.Stdout, f.entries); formatErr != nil {
+			logrus.Error(formatErr)
 		}
 	}
-	if len(errs) > 0 {
-		return fmt.Errorf("%d errors: %v", len(errs), errs)
-	}
-
-	return formatter.FormatSlice(options.Format, options.Stdout, f.entries)
+	return err
```

Comment:
behavior change:

old: walk all and return all errors if there is any; if no error, format entries.
new: walk all and _record_ all errors in `err`; format entries if > 0 and log format error if there is; return all errors from walk.

I think the new behavior more matches docker behavior: https://github.com/docker/cli/blob/e92dd87c3209361f29b692ab4b8f0f9248779297/cli/command/inspect/inspector.go#L70-L93